src/app/util/ember-compatibility-functions.cpp

```diff
@@ -134,6 +136,15 @@ EmberAfAttributeType BaseType(EmberAfAttributeType type)
     }
 }
 
+Protocols::InteractionModel::Status CheckAccessControl(const SubjectDescriptor & aSubjectDescriptor, EndpointId aEndpoint, ClusterId aCluster, bool aWrite)
+{
+    RequestPath requestPath = { .cluster = aCluster, .endpoint = aEndpoint };
+    // TODO: get required privilege using ember APIs, as it could be custom
+    Privilege privilege = aWrite ? Privilege::kOperate : Privilege::kView;
```

Comment:
Shouldn't this be retrieved from the specified privilege for the given attribute being read/written to?

Reply:
That's what the TODO says.